.circleci/config.yml

```diff
@@ -247,6 +253,9 @@ jobs:
       - run:
           name: Opcode tests
           command: yarn --cwd packages/protocol check-opcodes
+      - node/install:
+          install-yarn: true
+          node-version: '12.22.11'
```

Comment:
This was required only for this test.

This test generates a devchain with the latest deployed contracts(CR9), by checking out the latest `RELEASE_TAG` branch. The generated devchain is then used in `protocol-test-release` for version checks and backward compatibility.

Since the CR9 branch uses node12, we have to use node12 for this test, otherwise, the build fails.

This should only be done during this transitionning period, as the next contract release would be using node18.

I'll add a `TODO` to remove this once CR10 is deployed using node18